src/server/api/auxiaProxyRouter.ts

```diff
@@ -0,0 +1,156 @@
+import express, { Router } from 'express';
+import { getSsmValue } from '../utils/ssm';
+
+interface AuxiaApiRequestPayloadContextualAttributes {
+    key: string;
+    stringValue: string;
+}
+
+interface AuxiaApiRequestPayloadSurface {
+    surface: string;
+    minimumTreatmentCount: number;
+    maximumTreatmentCount: number;
+}
+
+interface AuxiaAPIRequestPayload {
+    projectId: string;
+    userId: string;
+    contextualAttributes: AuxiaApiRequestPayloadContextualAttributes[];
+    surfaces: AuxiaApiRequestPayloadSurface[];
+    languageCode: string;
+}
+
+interface AuxiaAPIAnswerDataUserTreatment {
+    treatmentId: string;
+    treatmentTrackingId: string;
+    rank: string;
+    contentLanguageCode: string;
+    treatmentContent: string;
+    treatmentType: string;
+    surface: string;
+}
+
+/*
+    user treatments example:
+    {
+        "treatmentId": "105867",
+        "treatmentTrackingId": "105867_336_b352c123-c298-493c-a794-90852ec24961",
+        "rank": "2",
+        "contentLanguageCode": "en-GB",
+        "treatmentContent": "{\"title\":\"Create a free account\",\"body\":\"\",\"first_cta_name\":\"Register\",\"first_cta_link\":\"https://profile.theguardian.com/signin?\",\"second_cta_name\":\"Next time\",\"second_cta_link\":\"https://profile.theguardian.com/signin?\",\"subtitle\":\"\"}",
+        "treatmentType": "DISMISSABLE_SIGN_IN_GATE",
+        "surface": "ARTICLE_PAGE"
+    }
+*/
+
+interface AuxiaAPIAnswerData {
+    responseId: string;
+    userTreatments: AuxiaAPIAnswerDataUserTreatment[];
+}
+
+interface AuxiaProxyResponseData {
+    shouldShowSignInGate: boolean;
+}
+
+const buildAuxiaAPIRequestPayload = async (): Promise<AuxiaAPIRequestPayload> => {
+    const projectId = await getSsmValue('PROD', 'auxia-projectId');
```

Comment:
Let's make sure all config is loaded once, in the same place, on startup only (not per request)

Reply:
Done!